robot_name: summit_xl
robot_config:
    mode: skid_steering
    wheel_diameter: 0.25
    track_dist: 1
    wheel_base: 0.446
    track_width: 0.408
robot_control:
    xicr: 0.02
    C: 2
    k_1: 1.2
    k_2: 1.5
    v_max: 0.1
